crates/clvm-traits/src/clvm_encoder.rs

```diff
@@ -43,6 +63,10 @@ impl ClvmEncoder for Allocator {
     ) -> Result<Self::Node, ToClvmError> {
         self.new_pair(first, rest).or(Err(ToClvmError::OutOfMemory))
     }
+
+    fn encode_bigint(&mut self, number: BigInt) -> Result<Self::Node, ToClvmError> {
+        self.new_number(number).or(Err(ToClvmError::OutOfMemory))
+    }
```

Comment:
this is nice. I envision that we may want an `encode_g1()` and `encode_g2()` at some point in the future as well (when the allocator supports those types natively)